app/select.js

```diff
@@ -0,0 +1,52 @@
+define(['app/database', 'jquery', 'parsley', 'bootstrap-select'], function (db, $, parsley, select) {
+  return {
+    initiate: function(name, item) {
+      switch (name) {
+        case 'languages':
+          // Load languages from database
```

Comment:
Use separate functions for the each of the cases. The only responsibility of this initiate functions should be to select the correct function call for each of the possible parameters.